src/modules/common/components/layout.jsx

```diff
@@ -3,13 +3,18 @@ import { connect } from 'react-redux';
 import SiteNav from './site-nav';
 import SiteFooter from './site-footer';
 import Landing from './landing';
+import AdminOnly from './admin-only';
```

Comment:
This isn't being used, but given my overall suggestion, this separate component wouldn't be needed at all.